jetbrains-core/tst-resources/toolkit-test-log.properties

```diff
@@ -1 +1 @@
-software.aws.toolkits.level=FINE
+software.aws.toolkits.level=FINEST
```

Comment:
What is this used for?

Reply:
was struggling to figure out what was happening